### DISCUSSION TITLE
关于集群的几个问题

### DISCUSSION THREAD

**Discussion by githubzhangfei** [dragonflydb/dragonfly#2804] [Q&A]
集群是否支持管道查询？据我所知redis是不支持的。我每秒数据1万条。这1万条数据。每条数据需要根据规则拆分，然后根据规则返回结果，使用管道的方式查询。管道中有大概120个左右的get。可以这样理解。qps1w   ,   TPS1万*120.不知道有没有人测试过。求分享

**Comment by romange** [comment]
huh?!
